### DISCUSSION TITLE
Using stellarium in Android system. Error message and frustration.

### DISCUSSION THREAD

**Discussion by ericclayt** [Stellarium/stellarium#2299] [Q&A]
Hi, and help please! I have been totally frustrated. Had Stellarium plus working happily on a Samsung Galaxy 7.
 Updated to a Samsung Galaxy S21 5G. It now wont work. There must be some obscure setting I have missed, though I had the the phones side by side and checked them. I am getting the Connection refused error. Even more frustrating is that when I open the application it will move the telescope and go to a spot. Then refuses to move again.
Any suggestions gratefully accepted. I know I could use it on the older phone but would like to use my current one.
Thanks again
Eric
The log is attached.
![Screenshot_mount1](https://user-images.githubusercontent.com/100329862/155452332-70cfb577-c975-4c27-8a70-9f0a107d08e6.jpg)

**Comment by xalioth** [comment]
Hi Erik,

thanks for the report, could you please contact us on the email dedicated
to issues in the mobile apps: ***@***.*** ?
Please specify which version of the app you are using, and also you
telescope setup, how do you connect it to the app ?

Fabien

**Comment by ericclayt** [comment]
Hi Fabien,
thank you for your quick reply. I am sorry I could not find the email dedicated to issues in the mobile app that you referenced. Would it be possible for you to email me at ericzeroclaytonatyahoodotcomdotau where you replace the zero by the number 0 
I am using version v1.8.5 stellarium mobile plus on a Samsung  S21 5G android system. The version that is working is v1.7.6 stellarium mobile plus running on a Samsung Galaxy S7.
My telescope is a skywatcher evostar. The mount is a skywatcher az-gte connected by wifi to my phone. This is a goto mount controlled by an app on the phone.
Hope you can help
Cheers Eric
